src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    static final int MAX_SIZE = 10;
+    private K[] keyArray;
+    private V[] valueArray;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```
we don't usually include types in the names of a variable - Java is a strongly typed language